src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,138 @@
+'use strict';
+
+const { getUserById } = require('../services/users.service');
+const {
+  getAllExpenses,
+  getExpenseById,
+  createExpense,
+  updateExpense,
+  removeExpense,
+} = require('./../services/expenses.service');
+
+async function getExpenses(req, res) {
+  const { userId, categories, from, to } = req.query;
+  let searchedExpenses;
+
+  try {
+    searchedExpenses = await getAllExpenses(
+      userId,
+      categories,
+      from,
+      to,
+    );
```

Comment:
```suggestion
  try {
    const searchedExpenses = await getAllExpenses(req.query);
```
- You don't need to use `let` in your case as you don't change this variable
- And you don't need to unpack data from a query as you pass the same into `getAllExpenses`